pulsar-client-kafka-compat/pulsar-client-kafka-tests/src/test/java/org/apache/pulsar/client/kafka/compat/examples/ConsumerExample.java

```diff
@@ -23,7 +23,7 @@
 
 import org.apache.kafka.clients.consumer.Consumer;
 import org.apache.kafka.clients.consumer.ConsumerRecords;
-import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.consumer.PulsarKafkaConsumer;
```

Comment:
I don't think this is the right fix. the examples are used to test the shaded pulsar kafka client wrapper. `PulsarKafkaConsumer` should already be shaded and relocated to `KafkaConsumer`. If this example doesn't work, it means the shading doesn't work properly or it has original kafka dependencies in the classpath.

If you are running on intellij, it might be a problem of intellij which doesn't handle shaded dependencies very well. so what I would suggest is to run the examples from command line to see if that's a problem or not.

Reply:
> I don't think this is the right fix. the examples are used to test the shaded pulsar kafka client wrapper. `PulsarKafkaConsumer` should already be shaded and relocated to `KafkaConsumer`. If this example doesn't work, it means the shading doesn't work properly or it has original kafka dependencies in the classpath.
> 
> If you are running on intellij, it might be a problem of intellij which doesn't handle shaded dependencies very well. so what I would suggest is to run the examples from command line to see if that's a problem or not.

Yes, example in IDEA USE original kafka KafkaConsumer, KafkaProducer. not shaded kafka.
if master [KafkaApiTest](https://github.com/apache/pulsar/blob/master/tests/pulsar-kafka-compat-client-test/src/test/java/org/apache/pulsar/tests/integration/compat/kafka/KafkaApiTest.java) Integration Test ok that is even more evident.

Reply:
I don't think you should change the constructor. but you can still change the topic to ` "persistent://public/default/test"`

Reply:
> The naming of shade coincides with the naming of native kafka, which leads to confusion

this was done intentionally. this jar can easily be used for swapping the existing kafka library without changing user application code.

Reply:
> this was done intentionally. this jar can easily be used for swapping the existing kafka library without changing user application code.

ok